waspc/src/Wasp/Analyzer/Parser/Valid.hs

```diff
@@ -0,0 +1,27 @@
+module Wasp.Analyzer.Parser.Valid
+  ( validateAst,
+  )
+where
+
+import Data.List (find)
+import qualified Wasp.Analyzer.Parser as P
+
+validateAst :: P.AST -> Either (String, P.Ctx) P.AST
+validateAst = validateNoEntityDeclInWaspFile
+
+validateNoEntityDeclInWaspFile :: P.AST -> Either (String, P.Ctx) P.AST
+validateNoEntityDeclInWaspFile ast@(P.AST stmts) = case findEntityDecl stmts of
```

Comment:
Why not `statements`?

(yes, I proudly admit I hate non-standard abbreviations).

Reply:
The data type is defined as:
```haskell
newtype AST = AST {astStmts :: [WithCtx Stmt]} deriving (Eq, Show)
```
so it seemed fitting to use `stmts` :D